server/server.js

```diff
@@ -2,12 +2,33 @@ const express = require('express')
 const app = express()
 const port = 3000
 
-const organizations = []
+const bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const organizations = {}
+let currId = 0
+
+
 
 app.get('/', (req, res) => res.send('Hello United Nepali!'))
 
 app.listen(port, () => console.log(`Example app listening on port ${port}!`))
 
 app.get('/organizations', (req, res) => res.send(organizations))
 
+app.post('/organizations', function(req, res){
+	let id_val = currId;
+	currId = currId + 1;
+
+	organizations[id_val] = {'Name':req.body.name};

```

Comment:
Nice I like the usage of a mapping over a array here!
I looks like you're trying to make the CREATE endpoint. I think maybe you may have missed the body spec from the CREATE issue:
https://github.com/nepalcodes/unitednepali/issues/9

I'm just mentioning it in this comment so that the PR is linked to it, but we can add the spec to the body later.